docs/admin-guide.md

```diff
@@ -0,0 +1,14 @@
+# Admin guide
+
+- Installation
+    - [Marketplace](#marketplace-installation)
+    - [Manual](#manual-installation)
+
+## Marketplace installation
```

Comment:
@Reene-Simon - Please remove this Marketplace installation section as it's no longer applicable/valid. Thanks!

Reply:
Sure👍